packages/pluggableWidgets/feedback-native/src/utils/sprintrApi.ts

```diff
@@ -1,36 +1,38 @@
-import { stringify } from "querystringify";
 import { Dimensions, Platform } from "react-native";
 
-const SPRINTR_FEEDBACK_URL = "https://sprintr.home.mendix.com/submitissue/";
+const SPRINTR_FEEDBACK_BASE_URL = "https://feedback.api-test.mendix.com/rest/v3/feedbackapi";
```

Comment:
Shouldn’t this url be the production one?

Reply:
@KevinVlaanderen

Reply:
the production API is not working yet, we use the test url for testing and it will be updated once the production API is fixed